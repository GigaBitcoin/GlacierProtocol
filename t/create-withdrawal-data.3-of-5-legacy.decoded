{
  "txid": "595ba2cf1a4ec1c9360ac309a2ca0ad853e71fd417b3e234bde479f60ef46cb7",
  "hash": "595ba2cf1a4ec1c9360ac309a2ca0ad853e71fd417b3e234bde479f60ef46cb7",
  "version": 2,
  "size": 481,
  "vsize": 481,
  "weight": 1924,
  "locktime": 0,
  "vin": [
    {
      "txid": "67848a7a2a6ebdad2d66aadb2e62aa84fe36ad9598c57569eb2b5a6abaa5f354",
      "vout": 2,
      "scriptSig": {
        "asm": "0 3045022100a990beb8f75c94d5b64f6361f164a5af4dc507df5067ebbffea5cf029747f74402202a001dcaeb13d3a76962e564f8cd0eef299616d47698d8ed69e1a7b7ef42bbf5[ALL] 30450221009ec967d2fcd3bfa960fc2517905ca506a4f668f971393f605b9bd53b82c7cac002207221036d4dc2b8e4d9698641385afb609fdc17a7761c09f02a4b8af0e6f29ad1[ALL] 304402204fdc33a365590ebaa352fd5fa82d07a32a873797453df15b32d12e864a05d8d50220312b9cbe51aebac7f14035ae94d15da5ede9682de8434a1046bf674b981805a5[ALL] 53210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d62103fa2d2c07653afb0a73340ee9e6b10ff0d4624c93b01127e9a69efbed48468e0121029669b79d7e05f20ca7aa418edc839da904467253c9e47194cb83e69ee418315155ae",
        "hex": "00483045022100a990beb8f75c94d5b64f6361f164a5af4dc507df5067ebbffea5cf029747f74402202a001dcaeb13d3a76962e564f8cd0eef299616d47698d8ed69e1a7b7ef42bbf5014830450221009ec967d2fcd3bfa960fc2517905ca506a4f668f971393f605b9bd53b82c7cac002207221036d4dc2b8e4d9698641385afb609fdc17a7761c09f02a4b8af0e6f29ad10147304402204fdc33a365590ebaa352fd5fa82d07a32a873797453df15b32d12e864a05d8d50220312b9cbe51aebac7f14035ae94d15da5ede9682de8434a1046bf674b981805a5014cad53210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d62103fa2d2c07653afb0a73340ee9e6b10ff0d4624c93b01127e9a69efbed48468e0121029669b79d7e05f20ca7aa418edc839da904467253c9e47194cb83e69ee418315155ae"
      },
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.19994870,
      "n": 0,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 b6c7733e23a271e57ca16e5cd885174917968db9 OP_EQUALVERIFY OP_CHECKSIG",
        "hex": "76a914b6c7733e23a271e57ca16e5cd885174917968db988ac",
        "reqSigs": 1,
        "type": "pubkeyhash",
        "addresses": [
          "mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99"
        ]
      }
    }
  ]
}
