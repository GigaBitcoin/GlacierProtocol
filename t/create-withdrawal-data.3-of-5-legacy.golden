Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
You will need to enter several pieces of information to create a withdrawal transaction.


*** PLEASE BE SURE TO ENTER THE CORRECT DESTINATION ADDRESS ***


Source cold storage address: 
Redemption script for source cold storage address: 
Destination address: 
How many unspent transactions will you be using for this withdrawal? 
Please paste raw transaction #1 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Transaction data found for source address.
TOTAL unspent amount for this raw transaction: 0.20000000 BTC

How many private keys will you be signing this transaction with? 
#: Key #1: Key #2: Key #3: 
Enter fee rate.
Satoshis per vbyte: 
Based on the provided rate, the fee will be 0.00005130 bitcoin.
Confirm? (y/n): 
Please enter the decimal amount (in bitcoin) to withdraw to the destination address.

Example: For 2.3 bitcoins, enter "2.3".

After a fee of 0.00005130, you have 0.19994870 bitcoins available to withdraw.

*** Technical note for experienced Bitcoin users:  If the withdrawal amount & fee are cumulatively less than the total amount of the unspent transactions, the remainder will be sent back to the same cold storage address as change. ***

Amount to send to mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99 (leave blank to withdraw all funds stored in these unspent transactions): 
Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.20000000 BTC in unspent supplied transactions
0 BTC going back to cold storage address 2N75fd3tacsz3g25Eak6XnP7wizd8scVnGe
0.19994870 BTC going to destination address mxBQD1QAYpwiudaCJdRhE9QSW9cokafJ99
Fee amount: 0.00005130

Signing with private keys: 
cQy39MvkMKeaEvG1wXejdtEvAdU381u9EBBmcKUJWbfn6QL82YvR
cUv81BVz1JWuA9uwECPvfsoyA1TqmFJBJaQUPYrxkdf44zQSiTZY
cVV1MBAgdAnrX8bjQWSJcgZAbVtprR8kgdfez858ejvXs3qZWqn2


Confirm? (y/n): 
Calculating transaction...


Sufficient private keys to execute transaction?
True

Raw signed transaction (hex):
020000000154f3a5ba6a5a2beb6975c59895ad36fe84aa622edbaa662dadbd6e2a7a8a846702000000fd8a0100483045022100a990beb8f75c94d5b64f6361f164a5af4dc507df5067ebbffea5cf029747f74402202a001dcaeb13d3a76962e564f8cd0eef299616d47698d8ed69e1a7b7ef42bbf5014830450221009ec967d2fcd3bfa960fc2517905ca506a4f668f971393f605b9bd53b82c7cac002207221036d4dc2b8e4d9698641385afb609fdc17a7761c09f02a4b8af0e6f29ad10147304402204fdc33a365590ebaa352fd5fa82d07a32a873797453df15b32d12e864a05d8d50220312b9cbe51aebac7f14035ae94d15da5ede9682de8434a1046bf674b981805a5014cad53210320849607eed265c09f51cda0e972426558276384870651250aa4e8638c334ced2102766c70472a8a3de751d126eddac8436338f8447b1967b5aa3b12a2b9a6e712e921039080ddc448e7fe4985676a7d2d6567dfc2febb8207a6f29b9737233f4c3b44d62103fa2d2c07653afb0a73340ee9e6b10ff0d4624c93b01127e9a69efbed48468e0121029669b79d7e05f20ca7aa418edc839da904467253c9e47194cb83e69ee418315155aeffffffff01f6183101000000001976a914b6c7733e23a271e57ca16e5cd885174917968db988ac00000000

Transaction fingerprint (md5):
334a7218a9f19ce8695a5e82d7f79fb6
QR code for transaction written to transaction.png
